Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1

```diff
@@ -631,16 +631,11 @@ function InvokeDTAExecHostExe([string] $Version, [System.Management.Automation.P
     $exePath = Join-Path -Path $vsRoot -ChildPath $ExeName
     $exePath = "'" + $exePath + "'"
     Try
-    {
-        $StartDate = New-Object -TypeName DateTime -ArgumentList:(2050,01,01)
-        $FormatHack = ($([System.Globalization.DateTimeFormatInfo]::CurrentInfo.ShortDatePattern) -replace 'M+/', 'MM/') -replace 'd+/', 'dd/'
-        $date1 = $StartDate.ToString($FormatHack)
-
-        $session = CreateNewSession -MachineCredential $MachineCredential
-        Invoke-Command -Session $session -ErrorAction Continue -ErrorVariable err -OutVariable out -scriptBlock { schtasks.exe /create /TN:DTAConfig /TR:$args[0] /F /RL:HIGHEST  /SC:ONCE /ST:23:59 ; schtasks.exe /run /TN:DTAConfig;  schtasks.exe /change /disable /TN:DTAConfig } -ArgumentList $exePath
-
-        Write-Verbose ("Error : {0} " -f ($err | out-string)) -Verbose
-        Write-Verbose ("Output : {0} " -f ($out | out-string)) -Verbose
+    {	   		
+		$session = CreateNewSession -MachineCredential $MachineCredential		
+        Invoke-Command -Session $session -ErrorAction SilentlyContinue -ErrorVariable err -OutVariable out -scriptBlock { schtasks.exe /create /TN:DTAConfig /TR:$args /F /RL:HIGHEST /SC:MONTHLY; schtasks.exe /run /TN:DTAConfig ; schtasks.exe /delete /F /TN:DTAConfig }  -ArgumentList $exePath		
```

Comment:
if we delete the task, we may end up loosing any debugging information about the history ... for ex. any error code or events. can we just disable it ?